### PR TITLE
EMIP Metadata new works

### PR DESCRIPTION
**Pull request by Ralph-Lee-UK**
Created new work records. I am confident that these will need some adjustments, and am grateful for any corrections you can make - all good lessons for me!

### CODE REVIEW

File: new/LIT6378HymnEstifanos.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6378HymnEstifanos" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እምጥበብከ፡ ስፉሕ፡ ወእምጸጋከ፡ ረቂቅ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">ʾƎmṭǝbabǝka sǝfuḥ waʾǝmṣǝggāka raqiq</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6378HymnEstifanos.xml:9]: Please separate the prepositions with -.

File: new/LIT6378HymnEstifanos.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                <title xml:lang="en" corresp="#t1">From your broad wisdom and from your slender grace</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6378HymnEstifanos.xml:10]: I would add ... at the end.

File: new/LIT6378HymnEstifanos.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6378HymnEstifanos" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እምጥበብከ፡ ስፉሕ፡ ወእምጸጋከ፡ ረቂቅ።</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6378HymnEstifanos.xml:8]: ጽ

File: new/LIT6378HymnEstifanos.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography"/><!--EMIP 764-->
+            <div type="edition">
+              <note>the text follows <ref type="mss" corresp="EMIP00764"/> ff. 71v-73r</note>
+              <div type="textpart" subtype="incipit" xml:lang="gez">
+                <ab>
+                  እምጥበብከ፡ ስፉሕ፡ ወእምጸጋከ፡ ረቂቅ።
```
Comment (DenisNosnitsin1970) [inline new/LIT6378HymnEstifanos.xml:60]: ጽ

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
I personally do not add rubrication in the work records, even if I follow the mss, and I correct the mistakes. All these features I record in individual witnesses.

File: new/LIT6376Suffering.xml [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                <foreign xml:lang="am">ሕማማተ፡ መስቀልስ፡ እንዴት፡ ነው፡ ቢሉ፡ ጌታ<add place="above">ች</add>ን፡ ኢየሲስ፡ ክርስቶስ፨ ስለቺ፡ ብሎ፡
+                  <hi rend="rubric">ተሰቀለ፡ ሞተ፡ ሐመ፡ በእንቲአነ፡ ሞተ፡ በእንቲአነ፡ ሞተ፡ በፈቃዱ፡</hi> ወተቀበረ፡ በሥምረቱ፡ እንዲሉ፡ ሊቃ<hi rend="rubric">ውንት፡</hi> መታመሙስ፡
+                  በሰውነቱ፡ ርስት፡ ነው፡ እንጂ፨</foreign></ab>
+              </div>
+              <div type="textpart" subtype="explicit" corresp="EMIP02338">
+              <ab><pb corresp="EMIP02338" n="42v" facs="046"/><cb n="b"/>
+                <foreign xml:lang="am"><hi rend="rubric">ዮሐንስ፡ ሆይ</hi>፡ ይህን፡ ነገር፡ እወቅ፡ ሐራስ፡ መርገም፡ ከሴት፡ ያደረሰ፡ ወንድ፡ ከወንድ፡ ያደረ<supplied reason="omitted">ሰ</supplied>ች፡ ሴት፡ ክፉም፡
+                  ሰው፡ ይህ፡ መጽሐፍ፡ ካለበት፡ ቦታ፡ አይድሱ። ደ<hi rend="rubric">ግሞ፡ ቢቻለቸው</hi>፡ ፴፫ቱን። ባይቻላቸው፡ ፭ቱን፡ የጥር፡ <hi rend="rubric">ማርያምን፡</hi>
+                  የሰኔ፡ <hi rend="rubric">ማርያምን፡</hi> የካቲት፡ <hi rend="rubric">ኪዳነ፡ ምሕረትን</hi>። በነሐሴ፡ ፲፮፡ ቀን፡ ፍ<hi rend="rubric">ልሰታን</hi>፡ የግንቦት፡
+                  <hi rend="rubric">ልደታን</hi>፡ እሊህን፡ በዓላቴን፡ያከበረ፡ ይህን፡ መጽሐፍ፡ የጻፈ፡ ያጻፈ፡ ያነበበ፡ የ</foreign>
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT6376Suffering.xml:70]: That makes sense, I will do this in the future, and correct this now.

[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT6376Suffering.xml]: Done

File: new/LIT6376Suffering.xml [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+                <pb corresp="EMIP02338" n="43r" facs="046"/><cb n="a"/>
+                <foreign xml:lang="am">
+                  ተተረጐመ፡ የሰማ፡ እሊህን፡ ሁሉ፡ በሞቱ፡ ጊዜ፡ እቆምላቸው፡ አለሁ፡ ብላ፡ ለ<hi rend="rubric">ዮሐንስ፡</hi> ነገረችው፡ ተፈጸመ፡ ራዕየ፡ <hi rend="rubric">ማርያም</hi>፡</foreign>
+                  <foreign xml:lang="gez">ዘከሠተ፡ ላቲ፡ ወልደ፡ ወአምላካ፡ በኂሩቱ፡ ወበሣህሉ፡ ለአምላክነ፡ ሎቱ፡ ስብሐት፡ ለዓለመ፡ ዓለም፡ አሜን።
+                  ኦእግዝእትየ፡ <hi rend="rubric">ማርያም</hi>፡ እቀብኒ፡ ወአድኀንኒ፡ እምኵሉ፡ እኩይ፡ ዘበሰማያት፡ ሊተ፡ ለግብረ፡ እግዚአብሔር፡ አነ፡ ኃጥእ፡ ወአባሲ፡ ወልደ፡ <hi rend="rubric">ጊዮርጊስ</hi>፡
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT6376Suffering.xml]: Done

File: new/LIT6377HymnMary.xml [hunk 7]
```diff
@@ -0,0 +1,79 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6377HymnMary" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እሰግድ፡ ለኪ፡ ጽጌ፡ ሐና፡ በብረከ፡ ሕሊና፡ ጥቡዕ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">ʿƎsagǝd laki ṣǝge Ḥanaā babǝraka ṭǝbuʿ</title>
+                <title xml:lang="en" corresp="#t1">I bow to you flower of Hanna, with the knees of a ready mind</title>
```
[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT6377HymnMary.xml:10]: Corrected

[the remark this answers is not in the document]
Comment (Ralph-Lee-UK) [reply new/LIT6377HymnMary.xml:10]: Meaning <gap reason="omitted"/>?

File: new/LIT6378HymnEstifanos.xml (hunk 1 above)
Comment (Ralph-Lee-UK) [reply new/LIT6378HymnEstifanos.xml:9]: OK. I think that I have done this correctly, with the - after the ወ as well

File: new/LIT6378HymnEstifanos.xml (hunk 2 above)
Comment (Ralph-Lee-UK) [reply new/LIT6378HymnEstifanos.xml:10]: As above, with <gap reason="omitted"/>?

File: new/LIT6378HymnEstifanos.xml (hunk 3 above)
Comment (Ralph-Lee-UK) [reply new/LIT6378HymnEstifanos.xml:8]: Done

File: new/LIT6378HymnEstifanos.xml (hunk 4 above)
Comment (Ralph-Lee-UK) [reply new/LIT6378HymnEstifanos.xml:60]: Done

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**